test/sql-tap/default.test.lua

```diff
@@ -66,7 +66,7 @@ test:do_catchsql_test(
 	);
 	]], {
 	-- <default-1.3>
-	1, "Failed to create space 'T3': default value of column 'Y' is not constant"
+    1, "Failed to create space 't3': default value of column 'y' is not constant"
```

Comment:
The problem here and below is that the test was written using tabs instead of spaces. Since we don't allow tabs, I used 4 spaces instead of tabs as described in the rules. However, github shows the tab length to be 8 spaces, so there was a visual problem. I do not think that it is necessary to fix this.

Reply:
No, I'm writing not about the visual problem (I can set the size of the tab to 4 spaces). I'm writing about the whitespace symbol difference (spaces instead of tab). However, if you decided to use spaces here (I believe, checkpatch.pl forced you), then OK.